detection-bot/src/agent.ts

```diff
@@ -0,0 +1,31 @@
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+
+export const NETHERMIND_ADDRESS: { [address: string]: boolean } = {
+  "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8": true,
+};
+
+let findingsCount = 0;
```

Comment:
This `findingsCount` variable doesn't seem to be used, and thus may be unnecessary.